packages/x-adapter-platform/src/types/responses/semantic-queries-response.model.ts

```diff
@@ -0,0 +1,7 @@
+import { PlatformSemanticQuery } from '../models';
+
+export interface PlatformSemanticQueriesResponse {
+  data: {
```

Comment:
Is not missing here the query `property` in the response?

https://semantics-api.internal.test.empathy.co/search_single/kroger?q=firecrackers&lang=en&filter_ids=NOT_PARTIAL%2CNOT_ALL_WORDS&term_merge=false&max_dist=0.6